play_tik_tak_toe.py

```diff
@@ -2,13 +2,14 @@
 
 # Global Variables
 isXTurn = True
+grid = "_1_|_2_|_3_\n_4_|_5_|_6_\n 7 | 8 | 9 "
 
 # Functions
 
-def PrintEmptyBoard():
-    print("___|___|___")
-    print("___|___|___")
-    print("   |   |   ")
+# def PrintEmptyBoard():
```

Comment:
why comment this out? why not just delete it?

Reply:
so that we can see it directly in the code and not try it that way again. i wouldnt go back through version history so this is the best reminder for me

Reply:
uhh, okay. I don't this this is the best way but I guess it's not harmful. How about we add a `TODO (BROOKLYN): Delete later` as a compromise?